packages/client/src/utils/referenceApi.ts

```diff
@@ -203,17 +204,25 @@ export async function importConditionals(): Promise<LoadConditionalsResponse> {
   return conditionals
 }
 
+type InjectedUtilities = {
+  intl: IntlShape
+}
+
 export type LoadHandlebarHelpersResponse = Record<
   string,
-  Handlebars.HelperDelegate
+  (injectedUtilities: InjectedUtilities) => Handlebars.HelperDelegate
 >
-async function importHandlebarHelpers(): Promise<LoadHandlebarHelpersResponse> {
-  // https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
-  const handlebars = await import(
-    /* @vite-ignore */ `${window.config.COUNTRY_CONFIG_URL}/handlebars.js`
-  )
 
-  return handlebars
+async function importHandlebarHelpers(): Promise<LoadHandlebarHelpersResponse> {
+  try {
+    // https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
+    const handlebars = await import(
+      /* @vite-ignore */ `${window.config.COUNTRY_CONFIG_URL}/handlebars.js`
+    )
+    return handlebars
+  } catch (error) {
+    return {}
```

Comment:
Definitely a big anti-pattern here. No sense to propagate the error upwards until we have a more detailed error than what `import` currently gives us

Reply:
Yeah would love to find a solution for error handling the imports :/ But should we throw an error anyway? Isn't the risk here that the certificates get unusable and there's not a visible reason for anyone why that's so

Reply:
Good point. That's a better default.